front/public/locales/fr/operationalStudies/home.json

```diff
@@ -1,5 +1,8 @@
 {
+  "confirmDeleteMessage_one": "Voulez-vous réellement supprimer le projet ?",
```

Comment:
The corresponding translations are missing in `en/operationalStudies/home.json`.

Reply:
Completed, thx